script.js

```diff
@@ -0,0 +1,108 @@
+let selectedLi;
+let target = '';
+let ul = document.querySelector('.page__levels');
+let button = document.querySelector('.page__button');
+let startGame = document.querySelector('.page');
+let pageGame = document.querySelector('.game');
+let gameCards;
+let gameCardInners;
+let active = document.querySelector('.active');;
+
+
+ul.onclick = function (event) {
+  target = event.target;
+  higthLight(target);
+}
+
+
+function higthLight(element) {
+  if (selectedLi) selectedLi.classList.remove('page__levels_element-active');
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/EmilLachinov8/JS-project/pullRequest?prid=7662088)
```suggestion
  if (selectedLi) {selectedLi.classList.remove('page__levels_element-active');}
```